core/src/main/scala/vault/op.scala

```diff
@@ -131,22 +131,25 @@ object Vault {
   ): VaultF[Unit] = Free.liftF(DeleteKubernetesRole(authClusterName, roleName))
 
   def createPKIRole(
+    pkiPath: Option[String],
```

Comment:
As a general comment, which we should change here, i generally add new arguments to a function at the end of the argument list as there are cases through the codebase where we sometimes rely on default params with multiple params of the same type (gotta love strings, for example). Whilst this is largely historical, it avoids accidentally changing functionality that relies on default params.